src/platform/plugins/shared/unified_search/public/query_string_input/query_bar_top_row.tsx

```diff
@@ -949,7 +937,15 @@ export const QueryBarTopRow = React.memo(
               justifyContent={shouldShowDatePickerAsBadge() ? 'flexStart' : 'flexEnd'}
               wrap
             >
-              {props.showProjectPicker && renderProjectPicker()}
+              {props.showProjectPicker && (
+                <ProjectPicker
+                  projectRouting={props.projectRouting}
+                  onProjectRoutingChange={props.onProjectRoutingChange}
+                  wrappingContainer={(children: React.ReactNode) => (
+                    <EuiFlexItem grow={false}>{children}</EuiFlexItem>
+                  )}
+                />
```

Comment:
Is it necessary to pass this `wrappingContainer` prop to the `ProjectPicker`? Can we wrap the component here with the `EuiFlexItem` and remove this `wrappingContainer` prop from the Project Picker component?

```suggestion
                <EuiFlexItem grow={false}>
                  <ProjectPicker
                    projectRouting={props.projectRouting}
                    onProjectRoutingChange={props.onProjectRoutingChange}
                  />
                </EuiFlexItem>
```